components/schema-migrator/migrations/201908141300_initialize_schema.up.sql

```diff
@@ -0,0 +1,205 @@
+-- Runtime
+
+CREATE TYPE runtime_status_condition AS ENUM (
+    'INITIAL',
+    'READY',
+    'FAILED'
+);
+
+CREATE TABLE runtimes (
+    id uuid PRIMARY KEY,
+    tenant_id uuid NOT NULL,
+    name varchar(256) NOT NULL,
+    description text,
+    status_condition runtime_status_condition DEFAULT 'INITIAL' ::runtime_status_condition NOT NULL,
+    status_timestamp timestamp NOT NULL,
+    auth jsonb
+);
+
+ALTER TABLE runtimes
+    ADD CONSTRAINT runtime_id_name_unique UNIQUE (tenant_id, name);
+
+-- Application
+
+CREATE TYPE application_status_condition AS ENUM (
+    'INITIAL',
+    'UNKNOWN',
+    'READY',
+    'FAILED'
+);
+
+CREATE TABLE applications (
+    id uuid PRIMARY KEY,
+    tenant_id uuid NOT NULL,
+    name varchar(36) NOT NULL,
+    description text,
+    status_condition application_status_condition DEFAULT 'INITIAL' ::application_status_condition NOT NULL,
+    status_timestamp timestamp NOT NULL,
+    healthcheck_url varchar(256)
+);
+
+ALTER TABLE applications
+    ADD CONSTRAINT application_id_name_unique UNIQUE (tenant_id, name);
+
+-- Webhook
+
+CREATE TYPE webhook_type AS ENUM (
+    'CONFIGURATION_CHANGED'
+);
+
+CREATE TABLE webhooks (
+    id uuid PRIMARY KEY,
+    tenant_id uuid NOT NULL,
+    app_id uuid REFERENCES applications (id) ON DELETE CASCADE NOT NULL,
+    url varchar(256) NOT NULL,
+    type webhook_type NOT NULL,
+    auth jsonb
+);
+
+-- API Definition
+
+CREATE TYPE api_spec_format AS ENUM (
+    'YAML',
+    'JSON'
+);
+
+CREATE TYPE api_spec_type AS ENUM (
+    'ODATA',
+    'OPEN_API'
+);
+
+CREATE TABLE api_definitions (
+    id uuid PRIMARY KEY,
+    tenant_id uuid NOT NULL,
+    app_id uuid REFERENCES applications (id) ON DELETE CASCADE NOT NULL,
+    name varchar(256) NOT NULL,
+    description text,
+    group_name varchar(256),
```

Comment:
`group` is reserved word: https://www.postgresql.org/docs/current/sql-keywords-appendix.html